### PR TITLE
Update numpy.ipynb

### PR DESCRIPTION
**Pull request by RichaldoElias**

### CODE REVIEW

**Comment by adammichaelwood** [comment]
I can't open this notebook in colab. I get a JSON error.
Does this happen to anyone else, following the colab link above from DocsBot?

File: site/en/tutorials/load_data/numpy.ipynb [hunk 1]
```diff
@@ -120,6 +120,7 @@
         " \n",
         "import numpy as np\n",
         "import tensorflow as tf"
```
Comment (lamberta) [inline site/en/tutorials/load_data/numpy.ipynb]: Needs a "," at the end of line---causing a JSON formatting error

Comment (rosariopfernandes) [reply site/en/tutorials/load_data/numpy.ipynb]:
```suggestion
        "import tensorflow as tf",
```

**Comment by lamberta** [comment]
I think the other fixes were already merged into master. Closing since there's not much left in this PR. Thanks

**Pull request closed without merging**